servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -71,6 +77,237 @@
         }
       }
     },
+    "/$/Upgrade": {
+      "post": {
+        "operationId": "ClusterUpgrades_Start",
+        "description": "Start cluster upgrades",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "startClusterUpgrade",
+            "in": "body",
+            "description": "The upgrade of the start cluster",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StartClusterUpgrade"
+            }
+          },
+          {
+            "$ref": "#/parameters/timeout"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The cluster upgrade",
+            "schema": {
+              "type": "string"
```

Comment:
Is the entire response body just a string? Can you provide an example in the description about how the responseBody will look like?